bedrock/pocket/templates/pocket/save-to-pocket.html

```diff
@@ -13,7 +13,9 @@
   {{ css_bundle('pocket-features') }}
 {% endblock  %}
 
-{% block page_title %}Save to Pocket{% endblock %}
+{{ ftl('pocket-save-') }}
```

Comment:
Question: this string isn't used in any rendered section of the template, is it redundant?